src/components/BottomBanner/BottomBanner.component.tsx

```diff
@@ -27,7 +27,11 @@ export const BottomBanner = () => {
       <Typography variant="body2">
         Alpha 0.1: Deposit/withdraw stablecoins from Ethereum and mint/burn
         meta-stablecoins on Rootstock. Visit our{' '}
-        <Link href="https://discord.gg/5SEetbpq" color="primary">
+        <Link
```

Comment:
It's good that you are reading the code more carefully than us rn. You will probably find more small bugs like this one over time.